src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -110,17 +116,22 @@ public class CommandManager {
   static final String SONARLINT_BROWSE_TAINT_VULNERABILITY = "SonarLint.BrowseTaintVulnerability";
   static final String SONARLINT_SHOW_TAINT_VULNERABILITY_FLOWS = "SonarLint.ShowTaintVulnerabilityFlows";
   static final String SONARLINT_SHOW_SECURITY_HOTSPOT_FLOWS = "SonarLint.ShowSecurityHotspotFlows";
+  static final String SONARLINT_SUGGEST_FIX_COMMAND = "SonarLint.SuggestFix";
   static final List<String> SONARLINT_SERVERSIDE_COMMANDS = List.of(
     SONARLINT_QUICK_FIX_APPLIED,
     SONARLINT_SHOW_ISSUE_DETAILS_FROM_CODE_ACTION_COMMAND,
     SONARLINT_SHOW_RULE_DESC_COMMAND,
     SONARLINT_OPEN_STANDALONE_RULE_DESCRIPTION_COMMAND,
     SONARLINT_BROWSE_TAINT_VULNERABILITY,
-    SONARLINT_SHOW_TAINT_VULNERABILITY_FLOWS);
+    SONARLINT_SHOW_TAINT_VULNERABILITY_FLOWS,
+    SONARLINT_SUGGEST_FIX_COMMAND);
   // Client side
   static final String SONARLINT_DEACTIVATE_RULE_COMMAND = "SonarLint.DeactivateRule";
   static final String RESOLVE_ISSUE = "SonarLint.ResolveIssue";
   static final String SONARLINT_ACTION_PREFIX = "SonarQube: ";
+  public static final MessageActionItem RETRY_ACTION = new MessageActionItem("Retry");
+  public static final MessageActionItem SHOW_ISSUE_DETAILS_ACTION = new MessageActionItem("Show issue details");
+  public static final int AI_CODEFIX_ISSUE_MESSAGE_MAX_LENGTH = 40;
```

Comment:
Out of curiosity, where does this max length come from?

Reply:
From me :D Since the QuickFix text will include issue message, we did not want to show the full thing as it will be waaay longer than other actions and will seem out of place. So I chose this length as a reasonable limit (based on how it looks like in the UI).

Reply:
Thanks! Then I believe there should be a comment about it, so that we can check for the consequences if it needs to be changed in the future.